apps/theming-designer/src/components/ColorPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import * as stylesImport from './app.scss';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Card } from '@uifabric/react-cards';
+// import { Dropdown } from '../../../../packages/office-ui-fabric-react/lib/Dropdown';
+import { ThemeDesignerColorPicker } from './ThemeDesignerColorPicker';
+
+const styles: any = stylesImport;
+
+interface IColorPanelProps {
+  primaryColor?: string;
+  textColor?: string;
+  backgroundColor?: string;
+}
+
+interface IColorPanelState {
+  primaryColor?: string;
+  textColor?: string;
+  backgroundColor?: string;
+}
+
+export class ColorPanel extends React.Component<IColorPanelProps, IColorPanelState> {
+  constructor(props: any) {
```

Comment:
Try to avoid `any` type as much as possible to help with development and maintainability.

This `any` is `IColorPanelProps`.

Also may also want to consider picking the types for your state interface from props to keep them in-sync. Assuming both interfaces are necessary.